packages/elfutils.rb

```diff
@@ -28,12 +28,20 @@ class Elfutils < Autotools
   depends_on 'zlib' # R
   depends_on 'zstd' # R
 
-  pre_configure_options "CFLAGS+=' -Wno-error ' CXXFLAGS+=' -Wno-error '"
+  pre_configure_options "CFLAGS+=' -Wno-error -D_FORTIFY_SOURCE=0' CXXFLAGS+=' -Wno-error -D_FORTIFY_SOURCE=0'"
   configure_options "#{ARCH == 'i686' ? '--disable-libdebuginfod --disable-debuginfod' : ''} --enable-maintainer-mode --program-prefix='eu-'"
 
   def self.patch
+    downloader 'https://raw.githubusercontent.com/openwrt/openwrt/refs/heads/main/package/libs/elfutils/patches/009-fix-null-dereference-with-lto.patch', 'bd81d483ed5474fd7e87a27e4c961bf8670f76c45f5fe9a273cb2f11d8f44ffc'
+    system 'patch -Np1 -i 009-fix-null-dereference-with-lto.patch'
+    downloader 'https://raw.githubusercontent.com/openwrt/openwrt/refs/heads/main/package/libs/elfutils/patches/102-fix-potential-deref-of-null-error.patch', '4be7368570bf64d38d34ae8147946e0d3741103e3b4dd0000d4e5c228d16e352'
+    system 'patch -Np1 -i 102-fix-potential-deref-of-null-error.patch'
+
     return unless ARCH == 'i686'
 
+    # downloader 'https://raw.githubusercontent.com/openwrt/openwrt/refs/heads/main/package/libs/elfutils/patches/101-no-fts.patch', '8b568dfda4b1c4708d317dc8937d8de6a1298036309306c172a4ea26ca400763'
+    # system 'patch -Np1 -i 101-no-fts.patch'
+
```

Comment:
No filelist updates?

Reply:
Ignore... I'm working on this workflow...